### DISCUSSION TITLE
No server/localhost address

### DISCUSSION THREAD

**Discussion by ElleSimpsonEdin** [TheLastBen/fast-stable-diffusion#347] [General]
I've been trying to run this to train a model, but having no luck even launching.
The last line of the "Start stable-diffusion" only outputs: "^C" instead of an address to connect to.

**Comment by TheLastBen** [comment]
You're using 7GB model, free colab doesn't have enough ram for that, try the argument --lowram,  if you're lucky, it will work

**Comment by ElleSimpsonEdin** [comment]
I'm on a T4 on Colab Pro. 16gb should be fine no? I mean I can run this model fine on local on an 8gb RTX2070

**Reply by TheLastBen** [reply]
I'm talking about RAM not VRAM

**Reply by ElleSimpsonEdin** [reply]
So I take it 12 isn't enough.
Thanks for the help, I'll see if I can connect to a GCP session instead

**Reply by TheLastBen** [reply]
try --lowram (not lowvram)

**Reply by ElleSimpsonEdin** [reply]
> try --lowram (not lowvram)

I'm sorry about my ignorance, but where does this argument go? I keep getting "lowram not defined"

**Reply by TheLastBen** [reply]
add it at the last 3 lines in the last cell (click twice on the cell to show the code)

![lw](https://user-images.githubusercontent.com/110583491/199560878-03a78623-d1f0-4bab-8d7e-73a18cf80e34.JPG)

**Reply by ElleSimpsonEdin** [reply]
thank you!
